Modules/Core/QuadEdgeMesh/test/itkQuadEdgeMeshPolygonCellTest.cxx

```diff
@@ -101,6 +101,8 @@ itkQuadEdgeMeshPolygonCellTest(int, char *[])
   ITK_TEST_EXPECT_TRUE(pointId);
   ITK_TEST_EXPECT_TRUE(endId);
 
+  delete quadEdgeMeshPolygonCell;
+
```

Comment:
Thanks for fixing this memory leak, Jon. Looks like the memory leak was introduced by PR #4156 commit f7615d27ac61a453d622f476a333fee99838498b right? Specifically: https://github.com/InsightSoftwareConsortium/ITK/blob/f7615d27ac61a453d622f476a333fee99838498b/Modules/Core/QuadEdgeMesh/test/itkQuadEdgeMeshPolygonCellTest.cxx#L92-L102

Instead of declaring `quadEdgeMeshPolygonCell` as a raw pointer, I would suggest to just place the cell object on the stack:

    PolygonCellType quadEdgeMeshPolygonCell{};

Memory management is then performed automatically, no need to do any manual `delete`.  

My two cents, Niels

Reply:
Will change it later tonight.

Reply:
Done in 28678d.

Reply:
Thanks Jon! Can you then also please replace `quadEdgeMeshPolygonCell->` with `quadEdgeMeshPolygonCell.` ? (Replace these arrows by dots!)

Reply:
Done in db7dbeb. Thanks Niels.